Scoop/Controller/Ride Posting/InitialPostRideViewController.swift

```diff
@@ -0,0 +1,294 @@
+        setupTitleLines()
+        
+        nextAction = UIAction { _ in
+            guard let navCtrl = self.navigationController else {
+                return
+            }
+            
+            guard let arrival = self.arrivalTextField.text, !arrival.isEmpty,
+                  let departure = self.departureTextField.text, !departure.isEmpty
+            else {
+                self.presentErrorAlert(title: "Error", message: "Please complete all fields.")
+                return
+            }
+            
+            // TODO: Replace nextViewController with the actual next VC to be passed in
+            self.delegate?.didTapNext(navCtrl, nextViewController: nil)
+        }
+        
+        setupNextButton(action: nextAction ?? UIAction(handler: { _ in
+            return
+        }))
+        
+        setupTitleLabel()
+        setupStudentDriverButton()
+        setupTaxiButton()
+        setupDepartureTextField()
+        setupArrivalTextField()
+        setupLabels()
+    }
+    
+    private func setupTitleLabel () {
+        titleLabel.font = UIFont(name: "Rambla-Regular", size: 16)
+        titleLabel.text = "TRANSPORTATION METHOD"
+        titleLabel.accessibilityLabel = "preferred contact method"
+        titleLabel.textColor = .black
+        view.addSubview(titleLabel)
+        
+        titleLabel.snp.makeConstraints { make in
+            make.leading.equalToSuperview().inset(32)
+            make.top.equalToSuperview().inset(237)
+            make.height.equalTo(20)
+            make.trailing.equalToSuperview()
+        }
+    }
+    
+    private func setupStudentDriverButton() {
+        studentDriverButton.isSelected = true
+        studentDriverButton.setTitle("Student driver", for: .normal)
+        studentDriverButton.setTitleColor(.black, for: .normal)
+        studentDriverButton.titleLabel?.font = UIFont(name: "SFPro", size: 16)
+        studentDriverButton.titleLabel?.adjustsFontSizeToFitWidth = true
+        studentDriverButton.setImage(UIImage(systemName: "circle"), for: .normal)
+        studentDriverButton.setImage(UIImage(systemName: "circle.inset.filled"), for: .selected)
+        studentDriverButton.tintColor = .black
+        studentDriverButton.titleEdgeInsets = UIEdgeInsets(top: 0, left: 10, bottom: 0, right: 0)
+        studentDriverButton.imageEdgeInsets = UIEdgeInsets(top: 0, left: 0, bottom: 0, right: 10)
+        studentDriverButton.addTarget(self, action: #selector(studentDriverButtonToggle), for: .touchUpInside)
+        view.addSubview(studentDriverButton)
+        
+        studentDriverButton.snp.makeConstraints { make in
+            make.top.equalTo(titleLabel.snp.bottom).inset(-20)
+            make.leading.equalTo(titleLabel.snp.leading)
+            make.width.equalTo(180)
+        }
+    }
+    
+    private func setupTaxiButton() {
+        taxiButton.isSelected = false
+        taxiButton.setTitle("Taxi                  ", for: .normal)
```

Comment:
Is there a way you can use UIEdgeInsets rather than using white space?

Reply:
+1

Reply:
Yeah definitely don't rely on white space